plugins/BEdita/Core/src/Model/Entity/User.php

```diff
@@ -62,6 +62,8 @@ class User extends Profile
      * {@inheritDoc}
      */
     protected $_hidden = [
+        'created_by_user',
```

Comment:
Remove this `$_hidden` array like in `Profile`?
Probably only `password_hash` has to be added? In constructor?